components/list/README.md

```diff
@@ -1,46 +1,88 @@
 # Lists
 
-## d2l-list
-
-The `d2l-list` is the container to create a styled list of items using `d2l-list-item` or `d2l-list-item-button`. It provides the appropriate `list` semantics as well as options for displaying separators, etc.
-
-![List](./screenshots/list.png?raw=true)
+A list displays a collection of objects of the same type. A list is primarily used in order to help users navigate to a full-page representation of a single object, or to select several items and execute an action on them.
 
+<!-- docs: demo autoSize:false size:large -->
```

Comment:
I tried to use autoSize but with the lists the sizing comes a bit too late sometimes for the resizing logic in the demo snippet to work correctly. I tried adding timeouts and that did help but I kept having to increase it to capture all cases and so not using autoSize here just seemed safer